### PR TITLE
fix(DIA-162): misaligned view all button in articles rail

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [DIA-162] 

### Description

This PR fixes a misalignment in the section title on the artist screen

Group 1 (4)

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix misaligned view all button in articles rail - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[DIA-162]: https://artsyproduct.atlassian.net/browse/DIA-162?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fix misaligned view all button in articles rail - mounir)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7e6451a7b1e8ffeba3805af44fd04f6fa2613028
</p>

File: src/app/Components/Artist/Articles/Articles.tsx [hunk 1]
```diff
@@ -19,7 +19,13 @@ export const Articles: React.FC<Props> = ({ articles, artist }) => {
   return (
     <Flex>
       <Flex flexDirection="row" justifyContent="space-between">
-        <Text variant="sm-display">{`Artsy Editorial Featuring ${artistData.name}`}</Text>
+        <Flex maxWidth="80%">
```
Comment (olerichter00) [inline src/app/Components/Artist/Articles/Articles.tsx:22]: I believe it's better to use `flex={1}` here. The remaining 20% of the screen might not provide enough space for the "View All" button on small devices.

```suggestion
        <Flex flex={1} pr={1}>>
```

**Review by olerichter00 — CHANGES_REQUESTED** [review]

File: src/app/Components/Artist/Articles/Articles.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/app/Components/Artist/Articles/Articles.tsx:22]: makes sense

Comment (MounirDhahri) [reply src/app/Components/Artist/Articles/Articles.tsx]: done

Comment (olerichter00) [reply src/app/Components/Artist/Articles/Articles.tsx]: What do you think about adding `pr={1}` as well to get some space between the `...` and the "View All"?

**Review by olerichter00 — APPROVED** [review]

**Pull request merged**